segments/git_stash.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+
+
+def add_git_stash_segment():
+    stash_count = subprocess.check_output('git stash list | wc -l', shell=True).strip()
```

Comment:
Doing the line count in Python may be more efficient. Would be interested in seeing if that were true.